Cargo.toml

```diff
@@ -11,6 +12,9 @@ members = [
     "sdk",
 ]
 
+[workspace.package]
```

Comment:
I'm not yet 100% convinced that release-plz is ready for workspace versioning. Could we please do that in a separate PR? (FYI @cdmurph32 is submitting a PR to release-plz that I think will fix the underlying issue.)